### DISCUSSION TITLE
SearchBox without appbase

### DISCUSSION THREAD

**Discussion by cdavid15** [appbaseio/reactivesearch#1841] [Q&A]
This might be a daft question but does SearchBox only work if you are using an appbase.io backed Elasticsearch?

Seen the new component was released in 3.27.0 but when added to my app which connects directly to an on premise elastic install we get this error:

>Error: enableAppbase is required to be true when using SearchBox component.

The documentation doesn't refer anywhere to suggest it can't be used like the other components but thought I would seek clarification.

**Comment by cdavid15** [comment]
Anyone any ideas? Would this better posted on StackOverflow rather than the discussion boards here?

**Comment by siddharthlatest** [comment]
@cdavid15 Yes, Searchbox requires the use of [ReactiveSearch API](https://github.com/appbaseio/reactivesearch-api) server, although it is available in an open-source form.
